CHANGES.md

```diff
@@ -55,6 +62,15 @@ To be released.
 
 ### Behavioral changes
 
+ -  Changed `BlockChain<T>.FindNextHashes()` to return at most `count`
+    number of `BlockHash`es regardless of the result.
+    `BlockChain<T>`. [[#2581], [#2584]]
```

Comment:
Does it mean that `BlockChain<T>.FindNextHashes()` will return more hashes even after the branchpoint (to fit `count`)?